favorite_files.py

```diff
@@ -71,28 +83,34 @@ def fn(focus_view):
             else:
                 # Decend into group
                 value -= self.num_files
-                self.files = Favs.all_files(group_name=self.groups[value][0].replace("Group: ", "", 1))
+                self.group_name = self.groups[value][0].replace("Group: ", "", 1)
+                self.files = Favs.all_files(group_name=self.group_name)
```

Comment:
I'm passing group_name as an object variable so that it can be read in the new block, otherwise I had to repeat that expression in Favs.all_files()